App1/App1.Android/Properties/AndroidManifest.xml

```diff
@@ -7,6 +7,8 @@
 	<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
 	<uses-permission android:name="android.permission.CAMERA" />
 	<uses-permission android:name="android.permission.BLUETOOTH" />
+	<uses-permission android:name="android.permission.VIBRATE" />
+	<uses-permission android:name="android.permission.FLASHLIGHT" />
```

Comment:
Why do we need flashlight permission?

Reply:
I've matched the permissions used in our app's AndroidManifest.xml. Possibly for use in conjunction with the camera, i.e. flash?

Reply:
Probably for camera usage, yeah.